src/ssd/ssd-titlebar.c

```diff
@@ -252,18 +254,23 @@ static void
 update_visible_buttons(struct ssd *ssd)
```

Comment:
I'm not sure I got this correctly. What is a good app to test the resizing to a very small size with?

Reply:
xterm

Reply:
thanks, should be the same behavior now.